extras/systemd/dynflow-sidekiq@.service

```diff
@@ -4,9 +4,9 @@ Documentation=https://theforeman.org
 After=network.target remote-fs.target nss-lookup.target
 
 [Service]
-Type=simple
+Type=notify
 User=foreman
-TimeoutSec=90
+TimeoutSec=300
```

Comment:
On my machine (4cpus, 16gb of ram) the service failed to start in time. Foreman service uses 300 so I set it here as well.